backend/tracim_backend/applications/upload_permissions/controller.py

```diff
@@ -110,6 +111,23 @@ def disable_upload_permission(self, context, request: TracimRequest, hapic_data=
             request.current_workspace, hapic_data.path.upload_permission_id
         )
 
+    @hapic.with_api_doc(tags=[SWAGGER_TAG__WORKSPACE_UPLOAD_PERMISSION_ENDPOINTS])
+    @hapic.handle_exception(UploadPermissionNotFound, HTTPStatus.BAD_REQUEST)
```

Comment:
Just for note: A 404 could have been appropriate here.

Reply:
maybe, but we do not use 404 at all in all tracim (except "real" page not found exception).